packages/client/hmi-client/src/services/charts.ts

```diff
@@ -804,3 +805,179 @@ export function createInterventionChart(
 	}
 	return spec;
 }
+
+/// /////////////////////////////////////////////////////////////////////////////
+// Funman charts
+/// /////////////////////////////////////////////////////////////////////////////
+
+enum BoxType {
+	Satisfactory = 'Satisfactory',
+	Unsatisfactory = 'Unsatisfactory',
+	Ambiguous = 'Ambiguous',
+	ModelChecks = 'Model checks'
+}
```

Comment:
This feels like you are jamming two different ideas into the same enum and might be a point of confusion later on, or that it is not well-named.

Because funman has only Satisfactory/Unsatisfactory "boxTypes"  it is a bit confusing to introduce new things and only use parts of it and apply to things that are not really boxes.

If we are using this to encode colour then we should just say so, and rather than `modelChecks[i].boxType` it may be clearer just to say `modelChecks[i].colour` or something.

Reply:
that makes sense thanks